src/frontend/apps/desk/src/features/menu/Menu.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { Box } from '@/components/';
+import useCunninghamTheme from '@/cunningham/useCunninghamTheme';
+
+import MenuItem from './MenuItems';
+import IconRecent from './assets/icon-clock.svg';
+import IconContacts from './assets/icon-contacts.svg';
+import IconGroup from './assets/icon-group.svg';
+import IconSearch from './assets/icon-search.svg';
+import IconFavoris from './assets/icon-stars.svg';
+
+export const Menu = () => {
+  const { colorsTokens } = useCunninghamTheme();
+  const { t } = useTranslation();
+
+  return (
+    <Box
+      as="menu"
+      className="m-0 p-0"
+      $background={colorsTokens()['primary-800']}
+      $height="100%"
+      $width="60px"
```

Comment:
I am not very familiar with styled component, but is it acceptable to have px values in your code? It's a general good practice to use rem values? Wdyt?

Reply:
https://github.com/numerique-gouv/people/pull/19/commits/21d6524cd4a615e75a7776e66ee2c564dff0334d#diff-82c3f333ee8669eeca15913247ade54ae5c756a380c35fa8c560733268ff6e7bR24

Styled component is just like a wrapper around your component to inject inside the css, so all the css works actually.

Reply:
Yeah I know all the CSS works, but usually in CSS we avoid giving pixel values to our elements and prefer rem?

Reply:
https://github.com/numerique-gouv/people/commit/21d6524cd4a615e75a7776e66ee2c564dff0334d#diff-82c3f333ee8669eeca15913247ade54ae5c756a380c35fa8c560733268ff6e7bR24